frame/society/src/lib.rs

```diff
@@ -1108,7 +1108,14 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 		const MAX_BID_COUNT: usize = 1000;
 
 		match bids.binary_search_by(|bid| bid.value.cmp(&value)) {
-			Ok(pos) | Err(pos) => bids.insert(pos, Bid {
+			// Insert new elements after the existing ones. This ensures new bids
+			// with the same bid value are further down the list than existing ones.
```

Comment:
i don't think it does - it only ensures that it's below at least *one* of the existing ones. we probably need to skip forward from the position until it gets to the end or the value changes, and then insert.

Reply:
I updated this, but not sure if it is done ergonomically.